src/ui/components/CreateIdentifier/components/IdentifierStage3.tsx

```diff
@@ -38,9 +33,7 @@ const IdentifierStage3 = ({
   // We'll need to work out a proper way to get 'ourIdentifier'.
   const identifiersData = useAppSelector(getIdentifiersCache);
   const CREATE_IDENTIFIER_BLUR_TIMEOUT = 250;
-  const ourIdentifier = identifiersData.find(
-    (identifier) => identifier.method === IdentifierType.KERI
-  )?.id;
+  const ourIdentifier = identifiersData[0]?.id;
```

Comment:
same here - maybe we can log something in case `identifiersData` is `[]`

Reply:
actually, this is being removed in the new multi-sig flow so ignore this comment.